examples/src/sweat.rs

```diff
@@ -0,0 +1,93 @@
+use serde_json::json;
+
+use near_sdk::json_types::{U128, U64};
+use workspaces::prelude::*;
+
+const SWEAT_WASM_FILEPATH: &str = "./target/wasm32-unknown-unknown/release/sweat.wasm";
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    // can't use sandbox on M1 because of
+    // https://github.com/near/workspaces-rs/issues/110
+    // ☹️
+    let worker = workspaces::testnet();
```

Comment:
@intmainreturn00 0.2.0 of workspaces has some minor breaking changes. Making the following change should resolve the errors:
```suggestion
    let worker = workspaces::testnet().await?;
```